scripts/create_tarred_transformer_lm_dataset.py

```diff
@@ -0,0 +1,303 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
This scripts folder is getting crowded. Maybe we should organize them based on tasks or collections. We mostly keep the scripts of nlp in their own example folder, under a subfolder named 'data'. But both locations are OK, your choice. Is this script can be used with the transfromer_lm.py example?

Reply:
Reorganizing scripts should be done in another PR, its out of scope of this one. Scripts folder should probably be refactored to emulate the current collections hierarchy so that can be tackled in another PR.